### ISSUE TITLE
us_ny_scraper: no tasks or ingest info

### ISSUE THREAD

**Issue by colincadams** [Recidiviz/recidiviz-data#483]
```
worker/work : Queue us-ny-scraper, processing task (_generic_scrape) for us_ny.
base_scraper/_generic_scrape : Scraping data for us_ny and endpoint: None
us_ny_scraper/populate_data : Data extraction did not produce a single person, as it should
app/log_exception : Exception on /scraper/work [POST]
Traceback (most recent call last):
  File "/env/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/env/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/srv/recidiviz/utils/auth.py", line 98, in auth_and_call
    return func(*args, **kwargs)
  File "/srv/recidiviz/ingest/worker.py", line 89, in work
    result = scraper_task(params)
  File "/srv/recidiviz/ingest/base_scraper.py", line 178, in _generic_scrape
    'IngestInfo must be populated if there are no more tasks')
ValueError: IngestInfo must be populated if there are no more tasks
```

**Comment by jamwalla** [comment]
Have we seen this more recently? I saw some similar issues that I introduced in #495 and fixed in #507 - I added `__bool__` to `IngestInfo` and then needed to change the DataExtractor to check `if ingest_info is None`. My bad if this is left over from that.

**Comment by colincadams** [comment]
It looks like we still got some last night: https://pantheon.corp.google.com/errors/CPed6sb5gq3wsQE?time=P1D&project=recidiviz-123&supportedpurview=project

Were those changes deployed?

**Comment by arian487** [comment]
Fixed in #609
